pom.xml

```diff
@@ -22,96 +18,26 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <groupId>com.jcabi</groupId>
-    <artifactId>parent</artifactId>
-    <version>0.62.0</version>
-  </parent>
-  <groupId>org.eolang</groupId>
-  <artifactId>dedup</artifactId>
-  <version>1.0-SNAPSHOT</version>
-  <packaging>jar</packaging>
-  <name>dedup</name>
-  <description>Specialization of Objects in EO Programs</description>
-  <url>https://github.com/objectionary/dedup</url>
-  <inceptionYear>2022</inceptionYear>
-  <organization>
-    <name>EO</name>
-    <url>https://github.com/objectionary/dedup</url>
-  </organization>
-  <licenses>
-    <license>
-      <name>MIT</name>
-      <url>https://www.eolang.org/LICENSE.txt</url>
-      <distribution>site</distribution>
-    </license>
-  </licenses>
-  <developers>
-    <developer>
-      <id>1</id>
-      <name>Yegor Bugayenko</name>
-      <email>yegor256@gmail.com</email>
-      <organization>yegor256.com</organization>
-      <organizationUrl>https://www.yegor256.com</organizationUrl>
-      <roles>
-        <role>Architect</role>
-        <role>Developer</role>
-      </roles>
-      <timezone>+3</timezone>
-    </developer>
-  </developers>
-  <issueManagement>
-    <system>GitHub</system>
-    <url>https://github.com/objectionary/dedup/issues</url>
-  </issueManagement>
-  <scm>
-    <connection>scm:git:git@github.com:objectionary/dedup.git</connection>
-    <developerConnection>scm:git:git@github.com:objectionary/dedup.git</developerConnection>
-    <url>https://github.com/objectionary/dedup</url>
-  </scm>
-  <ciManagement>
-    <system>rultor</system>
-    <url>https://www.rultor.com/s/dedup</url>
-  </ciManagement>
-  <distributionManagement>
-    <site>
-      <id>github-pages</id>
-      <url>https://github.com/objectionary/dedup</url>
-    </site>
-  </distributionManagement>
-  <dependencies>
-    <dependency>
-      <groupId>org.junit.jupiter</groupId>
-      <artifactId>junit-jupiter-api</artifactId>
-      <version>5.9.1</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.junit.jupiter</groupId>
-      <artifactId>junit-jupiter-params</artifactId>
-      <version>5.9.1</version>
-      <scope>test</scope>
-    </dependency>
-  </dependencies>
-  <profiles>
```

Comment:
@cahekska Maybe it's better to leave `qulice` plugin in the project? `Qulice` significantly helps to improve your code even before sending it to review. You can read more about it right [here](https://www.yegor256.com/2014/08/13/strict-code-quality-control.html)